packages/shared/types/staking.ts

```
@@ -454,3 +463,46 @@ export interface IEarnBabylonTrackingItem {
+    name: string;
+    version: string;
+    chainId: number;
+    verifyingContract: string;
+  };
+  message: {
+    owner: string;
+    spender: string;
+    value: string;
+    nonce: string;
+    deadline: string;
+    expiry? : string; // dai
```

ESLint

[error] Delete `·`

(prettier/prettier)